openapi-src/V2/paths/_index.yml

```diff
@@ -845,7 +845,7 @@
         name: MODEL
         in: path
         required: true
-        description: 'Currently only organisation, funding-programme, project-pitch, project, site, nursery, project-report, site-report, nursery-report, project-monitoring and site-monitoring are set up'
+        description: 'Currently only organisation, funding-programme, project, site, nursery, project-report, site-report, nursery-report, project-monitoring and site-monitoring are set up'
```

Comment:
I think this one shouldn't be removed. File uploading for project pitches will still go through this endpoint until Jose's current work is complete. It'll be up to him to remove this endpoint entirely at that time.